shoop/core/settings.py

```diff
@@ -136,3 +136,9 @@
 
 #: The submission URL for Shoop's telemetry (statistics) system
 SHOOP_TELEMETRY_URL = "https://telemetry.shoop.io/collect/"
+
+#: Default cache duration for various caches in seconds
+SHOOP_DEFAULT_CACHE_DURATION = 60 * 30
```

Comment:
Using bare ints rather than typed values has the disadvantage that it's harder to spot what value that magical 151200 is (compared to `timedelta(hours=42)`). We don't have to repeat Django's way of using bare numbers for time intervals here, timedelta has `total_seconds()` for passing the number of seconds to Django.